Ecommerce/index.css

```diff
@@ -1 +1,94 @@
-/* Write code below for ECOMMERCE TASK */
+/* Write code below for ECOMMERCE TASK */
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Welcome to My Shop</title>
+    <style>
+        body {
+            font-family: Arial, sans-serif;
+            margin: 0;
+            padding: 0;
+            background-color: #f5f5f5;
+            color: #333;
+        }
+        header {
+            background-color: #333;
+            color: #fff;
+            padding: 10px 20px;
+            text-align: center;
+            position: relative; /* Added position relative */
+        }
+        .brand-image {
+            position: absolute; /* Added position absolute */
+            top: 50%; /* Positioned vertically in the middle */
+            right: 20px; /* Positioned on the right side */
+            transform: translateY(-50%); /* Adjusting to vertically center */
+            max-width: 100px; /* Adjust size as needed */
+            height: auto;
+        }
+        .container {
+            max-width: 500px;
+            margin: 20px auto;
+            padding: 0 20px;
+        }
+        img {
+            max-width: 100%;
+            height: auto;
+            display: block;
+            margin: 20px auto;
+        }
+        form {
+            margin-top: 20px;

```

Comment:
Consider adding comments to describe the purpose of the form and its styles for easier maintenance in the future.